### DISCUSSION TITLE
Can't Upload ADMX

### DISCUSSION THREAD

**Discussion by ObiEdKenobi** [Weatherlights/Winget-AutoUpdate-Intune#38] [Q&A]
Whenever I try to upload the ADMX I receive the errror "Uploaded xml file with name WinGet-AutoUpdate-Configurator.admx is invalid.
 "
 
 Can @Weatherlights  or someone please assist?
 
 Edit: Disregard I was able to resolve it by downloading the Code as a zip and uploading that way instead.
 No clue how that worked and the other didn't but I'll take it

**Comment by Weatherlights** [comment]
Nice to hear that you where able to solve this yourself :).
